process_rsmas.py

```diff
@@ -12,69 +12,103 @@
 import os
```

Comment:
process_sentinel.py has been replaced by process_rsmas.py. changes here are from previous version which are renaming and a few lines added in the main to get the directories and reading template file.